src/components/TopBar/components/Search/Search.scss

```diff
@@ -9,21 +9,27 @@ $search-results-desktop-large-width-offset: layout-width(secondary, min) +
 .Search {
   position: absolute;
   z-index: z-index(search, $stacking-order);
-  top: top-bar-height();
-  left: $search-results-mobile-nav-toggle-offset;
-  display: flex;
-  align-items: stretch;
-  width: 100vw;
-  height: calc(100vh - #{top-bar-height() + spacing(tight)});
+  top: top-bar-height() - spacing();
```

Comment:
Might wanna store this (or just the `spacing()` part) in a variable to create clarity around how this calculation works.